draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1430,6 +1430,52 @@ A DCCP implementation maintains the maximum packet size (MPS) during operation o
+Selecting an appropriate method can allow MP-DCCP to realise different path utilization strategies that make MP-DCCP suitable for end-to-end implementation over the Internet or in controlled environments such as Hybrid Access or 5G ATSSS.
+
+### Path mobility {#path_mobility}
+
+The path mobility strategy provides the use of a single path with a seamless handover function to continue the connection when the currently used path is deemed unsuitable for service delivery.
+Some of the DCCP subflows of a MP-DCCP connection might become inactive due to either the occurrence of certain error conditions (e.g., DCCP timeout, packet loss threshold, RTT threshold, closed/removed) or adjustments from the MP-DCCP user.
+When there is outbound data to send and the primary path becomes inactive (e.g., due to failures) or de-prioritized, the MP-DCCP endpoint SHOULD try to send the data through an alternate path with a different source or destination address (depending on the point of failure), if one exists. This process SHOULD respect the path priority configured by MP_PRIO or if not available pick the most divergent source-destination pair from the original used source-destination pair.
+Note: Rules for picking the most appropriate source-destination pair are an implementation decision and are not specified within this document.
+Path mobility is supported in the current Linux reference implementation {{website}}.
+
+### Concurrent path usage
+Different to a path mobility strategy, the selection between MP-DCCP
+subflows is here a per-packet decision that is a part of the multipath
+scheduling process. This method would allow multiple subflows to be
+simultaneously used to aggregate the path resources to obtain higher
+connection throughput.
+
+In this scenario, the selection of congestion control, per-packet scheduling
+and potential re-ordering method determines a concurrent path utilization
+strategy and result in a particular transport characteristic.
+
+A concurrent path usage method uses a scheduling design that could seek to 
+maximize reliability, throughput, minimizing latency, etc.
+
+Concurrent path usage over the Internet can have implications. When a 
+Multipath DCCP connection uses two or more paths, there is no guarantee 
+that these paths are fully disjoint.  When two (or more) subflows share 
+the same bottleneck, using a standard congestion control scheme could 
+result in an unfair distribution of the capacity with the multipath 
+connection using more capacity than competing single path connections.  
+Multipath TCP uses the coupled congestion control Linked Increases 
+Algorithm (LIA) specified in {{RFC6356}} to solve this problem.  This 
```

Comment:
A final plea: I think if this spec becomes PS, we need to explicitly call-out any references to non-standards track documents.
/specified in/in the experimental specification/

Reply:
Solved in https://github.com/markusa/ietf-multipath-dccp/pull/190/commits/c49286059545b387ec7585d2c8849063866bdd4a